plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -119,19 +124,26 @@ public void done(IJobChangeEvent event) {
     }
   }
 
+  private String getConsoleName(String project) {
+    Date now = new Date();
```

Comment:
Google style strongly recommends using JodaTime instead of Date class. If all you're doing is showing the current time to the the user it probably doesn't matter that much. For anything else, use JodaTime.